stingray/spectroscopy.py

```diff
@@ -378,7 +398,9 @@ def get_phase_lag(cs, model):
     return cap_phi_1, cap_phi_2, avg_psi
 
 
-def compute_rms(spectrum, model, criteria="all"):
+def compute_rms(
+    spectrum: Union[Powerspectrum, Crossspectrum], model: Any, criteria: Optional[str]="all"
```

Comment:
Can use `astropy.modeling.core.Model` for the same reason as above

Reply:
I'll include these changes and set all instances of `model` to `astropy.modeling.core.Model`